### PR TITLE
#352 Change corretto version to install

### PR DESCRIPTION
**Pull request by dawn-minion**
Part of https://github.com/Proemion/psp-images/issues/352

### CODE REVIEW

File: bin/install [hunk 1]
```diff
@@ -63,13 +63,13 @@ get_java() {
 	local oracle_version=$(echo $version | sed -e 's/\./u/g')
 
 	local s3base=https://s3-eu-west-1.amazonaws.com/repo.proemion.com
-	local corretto_base=https://d2znqt9b1bc64u.cloudfront.net/amazon-$version-linux-x64.tar.gz
+	local corretto_base=https://corretto.aws/downloads/resources/$version/amazon-corretto-$version-linux-x64.tar.gz
 
 	case $version in
-		corretto-8.202.08.2)
+		8.242.08.1)
```
Comment (andrejserafim) [inline bin/install]:
```suggestion
		corretto-8.242.08.1)
```

File: bin/list-all [hunk 2]
```diff
@@ -1,3 +1,3 @@
-#!/bin/env bash
+#!/usr/bin/env bash
 
-echo "8.121 8.131 8.141 8.144 8.151 9.0.1 corretto-8.202.08.2"
+echo "8.121 8.131 8.141 8.144 8.151 9.0.1 8.242.08.1"
```
Comment (andrejserafim) [inline bin/list-all]:
```suggestion
echo "8.121 8.131 8.141 8.144 8.151 9.0.1 corretto-8.242.08.1"
```

**Comment by andrejserafim** [comment]
@dawn-minion when looking at this today the prefix made it very clear

**Review by unicolet — CHANGES_REQUESTED** [review]
@dawn-minion I would recommend to add the new version alongside the old one. If you want we can remove all others, as we don't use them anyways (AFAIK)

**Review by andrejserafim — APPROVED** [review]
@dawn-minion thanks!@

**Review by unicolet — APPROVED** [review]
@dawn-minion perfect, thanks!

**Pull request merged**